ycmd.el

```diff
@@ -1142,6 +1128,23 @@ Handle configuration file according the value of
     (ycmd--report-status 'unparsed)
     (ycmd-notify-file-ready-to-parse)))
 
+(defun ycmd--handle-error-exception (results)
+  "Handle exception and print message from RESULTS."
+  (let* ((msg (assoc-default 'message results))
+         (is-error (pcase msg
+                     ((or "Still no compile flags, no completions yet."
```

Comment:
These strings are from the completer classes from `ycmd`. Because some not so severe failures raise also `ValueError` or `RuntimeError` (i.e. no documentation available at point or cannot goto location), we filter here the ones that affect completion.